docs/src/mpc/examples.md

```diff
@@ -1,5 +1,5 @@
 # MPC Examples
-The MPC capability provided by `REopt` is essentially the optimal sizing and dispatch capability that `REopt` is commonly used for, but with the sizing problem removed. Also, the MPC model can be built for an arbitrary time length, or "horizon" (whereas a `REopt` model always includes an entire calendar year). The MPC model also requires the user to provide load and resource forecasts as inputs (whereas the typical `REopt` model allows one to use built-in load profiles as well as other API's such as PVWatts for the solar resource).
+The MPC capability provided by `REopt` is essentially the optimal sizing and dispatch capability that `REopt` is commonly used for, but with the sizing problem removed. Also, the MPC model can be built for an arbitrary time length, or "horizon" (whereas a `REopt` model always includes an entire calendar year). The MPC model also requires the user to provide load and resource forecasts as inputs (whereas the typical `REopt` model allows one to use built-in load profiles as well as other API's such as PVWatts for the solar resource). #TODO elaborate? 
```

Comment:
@NLaws were there other reasons why someone might want to use MPC instead of just fixing tech sizes? You had mentioned the solve time should be much quicker, right?

Reply:
It's really all about determining the optimal dispatch given the DER capacities and forecasts, (assuming that our "model" is correct for the "model predictive control")

Reply:
ok! Removed TODO in https://github.com/NREL/REopt.jl/pull/82/commits/3717d21b35cdfc3c944499f4892fb85fc289f0d6